nuxeo-coldstorage/src/test/java/org/nuxeo/coldstorage/service/TestDummyColdStorageService.java

```diff
@@ -76,6 +83,28 @@ protected String getBlobProviderName() {
         return "dummy";
     }
 
+    @Test
+    public void shouldBulkMoveToColdStorage() throws IOException {
+        List<DocumentModel> docs = new ArrayList<DocumentModel>();
+        for (int i = 0; i < 10; i++) {
+            docs.add(createFileDocument(DEFAULT_DOC_NAME, Blobs.createBlob(FILE_CONTENT + i)));
+        }
+        transactionalFeature.nextTransaction();
+        String query = "SELECT * FROM File";
+
+        BulkService bulkService = Framework.getService(BulkService.class);
+        String username = SecurityConstants.SYSTEM_USERNAME;
+        String commandId = bulkService.submit(
+                new BulkCommand.Builder(MoveToColdStorageContentAction.ACTION_NAME, query, username).build());
+        coreFeature.waitForAsyncCompletion();
+
+        BulkStatus status = bulkService.getStatus(commandId);
+        assertNotNull(status);
```

Comment:
status will never be null but you should assert that status.getState() is COMPLETED